packages/server/src/stages/rpc/index.ts

```diff
@@ -65,12 +65,14 @@ const isomorhicHandlerTemplate = (
 ) => `
 import {getIsomorphicRpcHandler} from '@blitzjs/core'
 const resolverModule = require('${resolverPath}')
-export default getIsomorphicRpcHandler(
-  resolverModule,
-  '${resolverPath}',
-  '${resolverName}',
-  '${resolverType}',
-) ${useTypes ? "as typeof resolverModule.default" : ""}
+export default resolverModule.default
+  ? getIsomorphicRpcHandler(
+      resolverModule,
+      '${resolverPath}',
+      '${resolverName}',
+      '${resolverType}',
+    ) ${useTypes ? "as typeof resolverModule.default" : ""}
+  : undefined
```

Comment:
I don't know why, but it seems `resolverModule` is an empty object on the front side so it doesn't work.

```suggestion
export default getIsomorphicRpcHandler(
  esolverModule,
  '${resolverPath}',
  '${resolverName}',
  '${resolverType}',
) ${useTypes ? "as typeof resolverModule.default" : ""}
```